package.json

```diff
@@ -7,6 +7,6 @@
     "license": "Apache-2.0",
     "main": "./lib",
     "name": "influx-udp",
-    "repository": "git://github.com/mediocre/node-influx-udp.git",
```

Comment:
I think that these changes should be in a separate PR, or should not exist at all. The metadata in `package.json` should only be changed by the releaser... unless you want to change the package name as well?

Given that we're doing a fork, I don't think we need to change the URL or version number. `npm install`-ing with a `git+https` URL in our repo should be sufficient.